website/lab/01/index.mdx

```diff
@@ -1050,6 +1348,9 @@ This table show the advantages and disadvantages of both methods.
 All you have to do is to make sure you are building a binary that does not depend on `std`, has no `main` function
 and provides an **empty panic handler**.
 
+<Tabs>
+
+<TabItem value="vscode" label="Visual Studio Code" default>
 To prevent `rust-analyzer` from showing an error on the first line and to automatically
```

Comment:
Place a link to *Editor Setup*. Make sure to state in the text that rust-analyser needs to be configured not to build all targets.

The `"allTargets": false` is written in a different way depending on the editor.